### PR TITLE
Feat/healthcare indicator 865

### LINKED ISSUE #865: Healthcare indicator
We come out with the idea that put a indicator likes the traffic lights , to show the health of each nodes endpoins

for example

Green: All the infomation is ok and up-date
Yellow: The node is working but the information is not updated
Red: The endpoin is dead

On this part of the webApp 
image

### PR DESCRIPTION
**Pull request by Torresmorah**
### What does this PR do?

- Resolve #865
- [X] Add node, node_info and endpoint table to the db model
- * Every 4 hours bp and node info is updated 
- * Every 2 minutes endpoint info is updated (response and head block time)
- [X] Implement node health check
- * If the status of the http request is 200 and the head_block_time is less than or equal to 3 minutes then endpoint is working
- * If the status of the http request is 200 the endpoint is outdated
- * If the status is not 200 there is a warning 
- * If there is no response, the endpoint is not working

### DB model

db

### Health Check Indicator

Temporary list of endpoints with the health indicator
imagen

Help modal with the meaning of icons
imagen

imagen

imagen

### Steps to test

1. Run the project locally
1. Go to hasura console
1. Check the new tables 

1. Go to endpoints pages
1. Check that health check is working as described.

### CODE REVIEW

**Comment by xavier506** [comment]
@Torresmorah  this looks awesome!

**Comment by fvives99** [comment]
Props to Helbertson, looks amazing 👏

File: webapp/src/components/EndpointsTable/index.js [hunk 1]
```diff
@@ -9,20 +10,53 @@ import TableHead from '@mui/material/TableHead'
...
+
 const EndpointsTable = ({ producers }) => {
+  const classes = useStyles()
   const { t } = useTranslation('endpointsListRoute')
 
-  const CellList = ({ producer, endpointType }) => {
+  const getStatus = (endpoint) => {
+    if (endpoint.response.status === undefined) return
 
+    if ((new Date() - new Date(endpoint.head_block_time)) / 60000 <= 3) {
```
Comment (xavier506) [inline webapp/src/components/EndpointsTable/index.js:27]: may be a good idea to set the time difference as a aparameter?

Comment (Torresmorah) [reply webapp/src/components/EndpointsTable/index.js:27]: I think this option is better, thanks

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**